stop_and_wait_plugin/src/stop_and_wait_plugin.cpp

```diff
@@ -79,6 +81,9 @@ namespace stop_and_wait_plugin
         ros::CARMANodeHandle::setSpinCallback([this]() -> bool
         {
             plugin_discovery_pub_.publish(plugin_discovery_msg_);
+            std_msgs::Float64 jerk_msg;
+            jerk_msg.data = jerk_;
+            jerk_pub_.publish(jerk_msg);
```

Comment:
Is this parameter constant?
This whole approach to setting the jerk value via a subscription seems both pretty computationally expensive assuming you are publishing a constant value each timestep, and overly complex. Why not just have both components load the same jerk ros parameter once at startup?

Reply:
I think the problem is that while the jerk is constant for a given trajectory, it may not be constant across all trajectories (other than as a max limit).

I think the best solution may be to expand the trajectory message to include metadata fields like the maneuver plan does. That way, we can make it part of the API contract for this control plugin that we can specify the jerk via those metadata fields. Shouldn't require too much work to add that, but I'm not sure if that data needs only go on the trajectory object itself or if it needs to go on each point. Putting it on each point gives us finer control, but bloats our message pretty substantially.